packages/@aws-cdk/aws-events-targets/lib/util.ts

```diff
@@ -27,9 +27,16 @@ export function singletonEventRole(scope: IConstruct, policyStatements: iam.Poli
  * Allows a Lambda function to be called from a rule
  */
 export function addLambdaPermission(rule: events.IRule, handler: lambda.IFunction): void {
+  let scope: Construct | undefined;
+  let node: ConstructNode = handler.permissionsNode;
+  if (rule instanceof Construct) {
```

Comment:
Deserves a comment explaining what we're doing here; obvious in the context of this PR, but possibly less so 3-4 months down the line.